src/Migration.php

```diff
@@ -0,0 +1,594 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration;
+
+use Exception;
+use Throwable;
+use Yiisoft\Db\Connection\Connection;
+use Yiisoft\Db\Exception\InvalidConfigException;
+use Yiisoft\Db\Exception\NotSupportedException;
+use Yiisoft\Db\Query\Query;
+use Yiisoft\Db\Schema\ColumnSchemaBuilder;
+use Yiisoft\Db\Schema\SchemaBuilderTrait;
+use Yiisoft\Strings\StringHelper;
+
+use function implode;
+
+/**
+ * Migration is the base class for representing a database migration.
+ *
+ * Migration is designed to be used together with the "vendor/bin/yii migrate" command.
+ *
+ * Each child class of Migration represents an individual database migration which is identified by the child class
+ * name.
+ *
+ * Within each migration, the {@see up() method should be overridden to contain the logic for "upgrading" the database;
```

Comment:
Seems that there are no up/down anymore. I'd propose renaming safeUp and safeDown to up and down.

Reply:
done.